code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -283,33 +283,33 @@
 	var/mob/living/carbon/victim = M
 	if(method == TOUCH || method == VAPOR)
 		//check for protection
-		var/mouth_covered = victim.is_mouth_covered()
 		var/eyes_covered = victim.is_eyes_covered()
-
+		
 		//actually handle the pepperspray effects
-		if ( eyes_covered && mouth_covered )
+		if (eyes_covered && istype(victim.wear_mask, /obj/item/clothing/mask/gas))
 			return
-		else if ( mouth_covered )	// Reduced effects if partially protected
+		else if(istype(victim.wear_mask, /obj/item/clothing/mask/gas))	// Reduced effects if partially protected
```

Comment:
prob should readd mouth_covered and just change it to this proc instead of `is_mouth_covered`

Reply:
Its supposed to be the sunglasses equivalent of pepperspray. Gas masks are, that is.